src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -342,4 +342,16 @@
   <data name="PX1054Title" xml:space="preserve">
     <value>A PXGraph instance should not start PXLongOperation during the initialization phase</value>
   </data>
+  <data name="PX1055Fix1" xml:space="preserve">
+    <value>Make the field bound to identity column the only key field in the DAC</value>
+  </data>
+  <data name="PX1055Fix2" xml:space="preserve">
+    <value>Alter the definition of the field bound to identity column to make it a non-key field and leave all other key fields untouched</value>
+  </data>
+  <data name="PX1055Fix3" xml:space="preserve">
+    <value>Remove the definition of the field bound to identity column and leave all other key fields untouched (it will be also very helpful to show a hint suggesting to drop the identity column, to which this key field is bound, from database)</value>
```

Comment:
@dropsonic , how I can do hint suggestion?

Reply:
@saloponov , consider the following (if the hind is displayed along with the fix message):
Remove the definition of the field bound to the identity column and leave all other key fields untouched. (You should drop the corresponding identity column from the database manually.)